### PR TITLE
Fs 2986 nstf config

### PR DESCRIPTION
**Pull request by Adam-W1**
https://digital.dclg.gov.uk/jira/browse/FS-2986

### Change description
Configure night shelter mapping in assessment 

Links to assessment store PR here: https://github.com/communitiesuk/funding-service-design-assessment-store/pull/194

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### Screenshots of UI changes (if applicable)
image
image

image

image

### CODE REVIEW

**Review by adamdavies1 — APPROVED** [review]
LGTM

File: app/assess/models/ui/applicants_response.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def _ui_component_from_factory(item: dict, application_id: str):
 
         return NewAddAnotherTable.from_dict(item)
 
-    elif presentation_type in ("text", "list"):
+    elif presentation_type in ("text", "list", "free_text"):
```
Comment (tferns) [inline app/assess/models/ui/applicants_response.py:256]: @ramsharma-prog You may need to revert this if the work you're doing to render the html is different FYI

Comment (Adam-W1) [reply app/assess/models/ui/applicants_response.py:256]: thanks Ram, i guess we merge your change to render free text and i can rebase onto it\?

Comment (ramsharma-prog) [reply app/assess/models/ui/applicants_response.py:256]: Sounds good @Adam-W1

Comment (ramsharma-prog) [reply app/assess/models/ui/applicants_response.py:256]: PR: https://github.com/communitiesuk/funding-service-design-assessment/pull/266/files

**Pull request merged**